components/UploadedItems.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+var items = 50;
+var finalURL = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&fl%5B%5D=identifier&fl%5B%5D=publicdate&fl%5B%5D=title&sort%5B%5D=publicdate+desc&sort%5B%5D=&sort%5B%5D=&rows=${items}&page=1&output=json`;
+
+class UploadedItems extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: 0
+    }
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  fetchData() {
+    fetch(finalURL)
+      .then((response) => response.json())
+      .then((responseJson) => {
+        const resultyt = responseJson.response.numFound;
+        this.setState({ result: (resultyt-1) });
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  render() {
+    return (
```

Comment:
Please use a better font and place the text in middle. Increase the font-size as well.
Use this text:
<a href="https://archive.org/details/@bub_wikimedia"><b>{No. of books}</b></a> uploaded to Internet Archive using BUB2!

Reply:
do I also need to add the link to the archive page of bub.wikimidia?

Reply:
Yes. This link: https://archive.org/details/@bub_wikimedia
That's what I added in the above comment.